src/charm.py

```diff
@@ -236,7 +236,10 @@ def create_scrape_job(config: JobConfig):
                 "static_configs": [
                     {
                         "targets": [config.target],
-                        "labels": {"node": kubernetes_snaps.get_node_name()},
+                        "labels": {
+                            "node": kubernetes_snaps.get_node_name(),
+                            "cluster": self.model.name,
```

Comment:
shouldn't this be the [cluster-name](https://github.com/charmed-kubernetes/interface-kube-control/blob/fa711adbfcfcd84a41440c50b3cb2f7c53488208/ops/ops/interface_kube_control/requires.py#L183)?  

if this is for the worker.. that's available from the kube-control relation